manifests/y/Youdao/YoudaoDict/10.0.12.0/Youdao.YoudaoDict.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-10
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Youdao.YoudaoDict
+PackageVersion: 10.0.12.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: /S
+  SilentWithProgress: /S
+Installers:
+- Architecture: x86
+  InstallerUrl: https://download.ydstatic.com/cidian/static/10.0/20230904/YoudaoDictSetup.exe
+  InstallerSha256: 922302C9BABCB55AC546017E52F1190EA85B9AAE6AEA80E340C4830C744A7C2
```

Comment:
```suggestion
  InstallerSha256: 922302C9BABCB555AC546017E52F1190EA85B9AAE6AEA80E340C4830C744A7C2
```

Reply:
Updated hash with downloaded file. I think the original hash wasn't long enough.

Reply:
you are right. The reason is I cannot paste hash text after copying files in github.dev.